pos_journal_image/views/templates.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+
+    <template id="pos_journal_image_js" inherit_id="point_of_sale.assets_backend">
```

Comment:
I think this will load the code from the backend and the frontend, I *think* the proper way to import this is:
<template id="assets_frontend" inherit_id="point_of_sale.assets"> 

@hparfr

Reply:
Done.